backend/daemon/storage/migrations.go

```diff
@@ -386,6 +386,45 @@ var migrations = []migration{
 			DELETE FROM kv WHERE key = 'last_reindex_time';
 		`))
 	}},
+	{Version: "2024-03-19.01", Run: func(_ *Dir, conn *sqlite.Conn) error {
+		return sqlitex.ExecScript(conn, sqlfmt(`
+		DROP TABLE IF EXISTS meta_view;
```

Comment:
This should probably be `DROP VIEW` not `DROP TABLE`.

Although, I would just use this complex query where it's needed, because it seems to be only needed in one place. Then we won't need any migrations, which is always a plus.